analysers/analyser_merge_fuel_IT.py

```diff
@@ -53,18 +54,7 @@ def __init__(self, config, logger = None):
                     static2 = {'source': self.source},
                     mapping1 = {
                         'ref:mise': 'idImpianto',
-                        'operator': lambda res: self.normalizeString(res['Gestore']),
+                        'operator': lambda res: italian_strings.normalize(res['Gestore']),
                         'brand': 'Bandiera'},
                 text = lambda tags, fields: {'en': u'%s, %s' % (fields['Indirizzo'], fields['Comune'])} )))
 
-    # First Char Uppercase
-    # quotes (") removal
-    # asterisk (*) removal
-    # extra spaces trim
-    # special case stopwords
-    WORDS_MAP = {'A': 'a', 'E': 'e', 'Ed': 'ed', 'Di': 'di', 'Dei': 'dei', 'In': 'in', 'Societa\'': 'Società', 'Sigla': 'sigla',
-        'Responsabilita\'': 'Responsabilità', 'Snc': 'S.N.C.', 'Sas': 'S.A.S.', 'S.P.A.': 'S.p.A.', 'Srl': 'S.R.L.', 'F.Lli': 'F.lli'}
```

Comment:
Please keep the analyser specific part here. Lot of string are specifics.

Reply:
In my opinion is better to keep it in a common place and share that code.. In the end, it will be helpful in all 3 merger and in this way we can do asserts to check it's all ok.
Are you worried about extra time to do the lookup?

Reply:
Some change for one can have impact on the other.

Reply:
Specific strings are not commons. But dedicated to this analysis.